performance-storage-service/pss_project/api/serializers/rest/metrics/MemoryMetricsSerializer.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework.serializers import Serializer, DecimalField
+from pss_project.api.models.rest.metrics.MemoryMetrics import MemoryMetrics, MemoryItemSummary
+
+
+class MemoryItemSummarySerializer(Serializer):
+    avg = DecimalField(max_digits=10, decimal_places=4, coerce_to_string=False)
```

Comment:
Are you sure the max_digits value is right? this means the max value is `999999.9999` because max_digits includes the decimal places. 

Also is there something in the script to ensure that we are rounding to 4 decimal places cause it will throw an error if it gets more than 4 decimal places.

Reply:
I removed the max limit since in the future there could be no upper limit for the memory consumption (maybe hardware bounded)